apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -459,25 +472,62 @@ private Beneficiary selectBeneWithLatestReferenceYear(List<Beneficiary> duplicat
     return entityManager.find(Beneficiary.class, maxReferenceYearMatchingBeneficiaryId);
   }
 
-  /** Enumerates the supported "should we include unique beneficiary identifiers" options. */
-  public static enum IncludeIdentifiersMode {
-    INCLUDE_HICNS_AND_MBIS,
+  /**
+   * The header key used to determine which header should be used. See {@link
+   * #returnIncludeIdentifiersValues(RequestDetails)} for details.
+   */
+  public static final String HEADER_NAME_INCLUDE_IDENTIFIERS = "IncludeIdentifiers";
 
-    OMIT_HICNS_AND_MBIS;
+  /**
+   * The List of valid values for the {@link #HEADER_NAME_INCLUDE_IDENTIFIERS} header. See {@link
+   * #returnIncludeIdentifiersValues(RequestDetails)} for details.
+   */
+  public static final List<String> VALID_HEADER_VALUES_INCLUDE_IDENTIFIERS =
+      Arrays.asList("true", "false", "hicn", "mbi", "");
```

Comment:
Why is the empty string allowed?

Reply:
This is for the case where headerValues is <blank>. I'll move this in to the related IF conditional to be more readable.

Reply:
TY! Just updated the code for that.